### PR TITLE
Fix translations

### PR DESCRIPTION
**Pull request by zorun**
Fix source strings for two translation issues (see commit messages for details)

### CODE REVIEW

**Comment by zorun** [comment]
Oops, I forgot to update tests. Will fix.

**Comment by zorun** [comment]
Tests should now be passing.

File: ihatemoney/translations/cs/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -739,3 +739,9 @@ msgstr ""
 msgid "Period"
 msgstr ""
 
+#~ msgid "%(member)s had been added"
+#~ msgstr ""
+
+#~ msgid "Person"
+#~ msgstr ""
```
Comment (Natim) [inline ihatemoney/translations/cs/LC_MESSAGES/messages.po:746]: I guess we can remove them.

Comment (zorun) [reply ihatemoney/translations/cs/LC_MESSAGES/messages.po]: Ah yes, right! Thanks for the review

**Comment by zorun** [comment]
@Natim thanks for merging!

I think it's useful to keep old strings (especially if they are translated in several languages) because Weblate can use them to suggest translations for new strings if they look similar.

**Comment by Natim** [comment]
Yeah I think weblate is less clever than tool I am used to. Such as Pontoon or Lingohub in that matter.
Especially regarding conflict resolution and keeping info in a local database.

**Comment by Natim** [comment]
I put them back in 64fbda8

**Comment by zorun** [comment]
Thanks!

**Pull request merged**